### PR TITLE
Added utility methods for optout in-app and push

### PR DESCRIPTION
**Pull request by ketanshikharebsft**
Added utility methods for enabling and disabling push and in-app messages

### CODE REVIEW

File: BlueShift-iOS-SDK/BlueShift.m [hunk 1]
```diff
@@ -899,6 +899,18 @@ - (BOOL)isTrackingEnabled {
     return [status isEqual:kYES] ? YES : NO;
 }
 
+#pragma mark Opt out push notifications
+- (void)optOutFromPushNotifications:(BOOL)isOptedOut {
+    [BlueShiftAppData currentAppData].enablePush = isOptedOut;
```
Comment (rahulrvp) [inline BlueShift-iOS-SDK/BlueShift.m:904]: I think this is incorrect. When you say you want to opt out, you will be passing YES/true here. If you set that to enablePush, it is basically an opt-in.

File: BlueShift-iOS-SDK/BlueShift.m [hunk 2]
```diff
@@ -899,6 +899,18 @@ - (BOOL)isTrackingEnabled {
     return [status isEqual:kYES] ? YES : NO;
 }
 
+#pragma mark Opt out push notifications
+- (void)optOutFromPushNotifications:(BOOL)isOptedOut {
```
Comment (rahulrvp) [inline BlueShift-iOS-SDK/BlueShift.m:903]: I think we can follow the same naming convention for `enableTracking`. We can provide the description in the method's comments/docs about what it does under the hood.

Suggestions are, `enablePushNotifications:(BOOL)isEnabled` or `optInForPushNotifications:(BOOL)isEnabled`

**Review by rahulrvp — CHANGES_REQUESTED** [review]

File: BlueShift-iOS-SDK/BlueShift.m (hunk 1 above)
Comment (ketanshikharebsft) [reply BlueShift-iOS-SDK/BlueShift.m:904]: Thanks for bringing it to notice, I will modify it based on the finalized method names.

**Review by rahulrvp — APPROVED** [review]
LGTM

File: BlueShift-iOS-SDK/BlueShift.m (hunk 2 above)
Comment (ketanshikharebsft) [reply BlueShift-iOS-SDK/BlueShift.m]: changed method names to `optInForPushNotifications ` format

**Pull request merged**